app/models/voucher/voucher.rb

```diff
@@ -20,7 +20,7 @@ class Voucher < ApplicationRecord
   DEFAULT_EXPIRATION_DAYS = 3
 
   ROLE_HASH = { tutor: 0, editor: 1, teacher: 2, speaker: 3 }.freeze
-  enum role: ROLE_HASH
+  enum :role, ROLE_HASH
```

Comment:
This violation should have been detected by RuboCop. Opened [an issue](https://github.com/rubocop/rubocop-rails/issues/1474) for it.

Reply:
This is really weird. I've created a new issue for it #768. For now, let's ignore this in this Rails upgrade that doesn't have to do with it and fix the RuboCop issue afterwards. Feel free to resolve this thread.